src/core/CMakeLists.txt

```diff
@@ -152,13 +152,19 @@ if(${TRITON_ENABLE_S3})
 endif()
 
 if(${TRITON_ENABLE_GPU})
-if ((NOT WIN32) AND (NOT ${ARCH} MATCHES "aarch64"))
+  set(HOST_COMPILER_FLAGS "")
+  if (WIN32)
+    set(HOST_COMPILER_FLAGS "/MD")
+  endif()
+
   set(CUDA_LIBRARIES PUBLIC ${CUDA_LIBRARIES})
   cuda_add_library(
     kernel-library 
     kernel.cu kernel.h
+    OPTIONS -arch compute_60
+    OPTIONS -code compute_60,sm_60,sm_61,sm_62,sm_70,sm_72,sm_75
```

Comment:
Ampere is sm_80 and sm_86.

Reply:
Some of the jetson machines have cuda compute capability under 6.0
Nano and TX1 have 5.3
We can either add them in optionally or customize this based on the min cuda compute flag from build.py

Reply:
Oh yes, add sm_53 as well.

Reply:
So lower the compute to 53?

I added sm_80 before but the Jetson build complains about it, it seems like it can't recognize it yet.

Reply:
Yes, it should be sufficient to use compute_53